includes/bash_profile

```diff
@@ -1,25 +1,22 @@
 # ~/.bash_profile: executed by bash(1) for interactive shells.
 
+source /opt/openhabian/functions/openhab.bash
+
 if [ -f "/opt/openHABian-install-failed" ]; then
-  echo ""
-  echo -e -n "\e[31;01mAttention! \e[39;49;00m"
+  echo -e -n "\\n\e[31;01mAttention! \e[39;49;00m"
   echo "The openHABian setup process seems to have failed on your system."
   echo "Sorry, this shouldn't happen! Please restart the installation process. Chances"
-  echo "are high setup will succeed on the second try."
-  echo ""
+  echo -e "are high setup will succeed on the second try.\\n"
```

Comment:
Didn't know about this text so far so not strictly related to your changes but we should fix this nonetheless.
Reword to explain *how* exactly to restart install. A beginner will not know and AFAIK that isn't even explained anywhere.
I think it should go into the debug guide.
Even I myself am not sure what to write, my current best best would be
`rm -f /opt/openHABian*; reboot`